app/services/LocationService.js

```diff
@@ -3,12 +3,22 @@ import PushNotificationIOS from '@react-native-community/push-notification-ios';
 import { NativeModules, Platform } from 'react-native';
 import PushNotification from 'react-native-push-notification';
 
-import { CROSSED_PATHS, PARTICIPATE } from '../constants/storage';
+import {
+  GOV_DO_TOKEN,
+  MEPYD_C5I_API_URL,
+  MEPYD_C5I_SERVICE,
+} from '../constants/DR/baseUrls';
+import {
+  COVID_POSITIVE,
+  CROSSED_PATHS,
+  PARTICIPATE,
+} from '../constants/storage';
 import { GetStoreData, SetStoreData } from '../helpers/General';
 import languages from '../locales/languages';
 
 let isBackgroundGeolocationConfigured = false;
 const LOCATION_DISABLED_NOTIFICATION = '55';
+const CODID_BASE_ID = '5590D7B3781E7592F6638F0D0D778282';
```

Comment:
don't you mean covid??
```suggestion
const COVID_BASE_ID = '5590D7B3781E7592F6638F0D0D778282';
```